src/App.vue

```diff
@@ -22,6 +24,12 @@ export default Vue.extend({
   created() {
     this.apps = require('./appregistry.json');
   },
+  methods: {
+    // TODO: REMOVE THIS REF AND METHOD WHEN VUEX IS ADDED
+    update() {
+      this.$refs.sidebar.refresh();
```

Comment:
You'll need to get around the typechecker here, by doing something like
```suggestion
      (this.$refs.sidebar as any).refresh();
```

Reply:
See 9ab3a2e